helm_deploy/hmpps-interventions-service/templates/ingress.yaml

```diff
@@ -33,7 +33,7 @@ spec:
     - host: {{ .host }}
       http:
         paths:
-          - path: /sent-referrals/summary/service-provider
+          - path: /sent-referrals
```

Comment:
This path will prefix match to:

- `/sent-referrals`
- `/sent-referrals/summary/service-provider`
- `/sent-referrals/paged`